### PR TITLE
Add link to the bandcamp profile in the social links part

### PR DESCRIPTION
**Pull request by just1602**

### CODE REVIEW

File: app/helpers/links_helper.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ module LinksHelper
     'Crimethinc. on Facebook'          => 'https://www.facebook.com/To-Change-Everything-103734818122357',
     'CrimethInc. on Github'            => 'https://github.com/crimethinc',
     'CrimethInc. on Tumblr'            => 'https://crimethinc.tumblr.com',
+    'Crimethinc. on bandcamp'          => 'https://crimethinc.bandcamp.com',
```
Comment (veganstraightedge) [inline app/helpers/links_helper.rb:7]:
```suggestion
    'CrimethInc. on Bandcamp'          => 'https://crimethinc.bandcamp.com',
```

File: app/helpers/links_helper.rb [hunk 2]
```diff
@@ -4,6 +4,7 @@ module LinksHelper
...
     'CrimethInc. on Telegram'          => 'https://t.me/ExWorkers',
     'Crimethinc. on TikTok'            => 'https://tiktok.com/@crimethinc',
     'Crimethinc. on Peertube'          => 'https://kolektiva.media/a/crimethinc',
```
Comment (veganstraightedge) [inline app/helpers/links_helper.rb:10]:
```suggestion
    'CrimethInc. on TikTok'            => 'https://tiktok.com/@crimethinc',
    'CrimethInc. on Peertube'          => 'https://kolektiva.media/a/crimethinc',
```

File: app/helpers/links_helper.rb [hunk 3]
```diff
@@ -19,6 +20,7 @@ def social_links_by_domain
       'Github'   => 'https://github.com/crimethinc',
       'Telegram' => 'https://t.me/ExWorkers',
       'Tumblr'   => 'https://crimethinc.tumblr.com',
+      'bandcamp' => 'https://crimethinc.bandcamp.com',
```
Comment (veganstraightedge) [inline app/helpers/links_helper.rb:23]:
```suggestion
      'Bandcamp' => 'https://crimethinc.bandcamp.com',
```

File: app/helpers/links_helper.rb (hunk 1 above)
Comment (just1602) [reply app/helpers/links_helper.rb:7]: I let `bandcamp` in lowercase because on their site, their logo is all in lower case.

**Review by veganstraightedge — APPROVED** [review]

File: app/helpers/links_helper.rb (hunk 1 above)
Comment (veganstraightedge) [reply app/helpers/links_helper.rb]: The way I always check is to look at the plaintext version in the `<title>` tag.

**Pull request merged**